go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/lib/pq v1.9.0
 	github.com/libp2p/go-libp2p-core v0.8.5
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
+	github.com/minio/sha256-simd v1.0.0
```

Comment:
Required due to v0 of specs-actors hamt https://github.com/filecoin-project/sentinel-visor/pull/519/commits/cf343479516df95cf1360cf7f99d1d19a87d70fc#diff-49836f46c8b64b9409ed3f44f1129c1b6bff9777bfc94f9b3d01bcf00ee5e4f8R131